source/change-streams/tests/unified/change-streams-resume-errorLabels.yml

```diff
@@ -0,0 +1,1342 @@
+                aggregate: *collection0
+                cursor: {}
+                pipeline:
+                  - $changeStream:
+                      resumeAfter:
+                        $$unsetOrMatches:
+                          $$exists: true
+              commandName: aggregate
+              databaseName: *database0
+  - description: change stream resumes after StaleShardVersion
+    runOnRequirements:
+      - minServerVersion: 4.3.1
+        topologies:
+          - replicaset
+          - sharded
+          - load-balanced
+    operations:
+      - name: failPoint
+        object: testRunner
+        arguments:
+          client: globalClient
+          failPoint:
+            configureFailPoint: failGetMoreAfterCursorCheckout
+            mode:
+              times: 1
+            data:
+              errorCode: 63
+              closeConnection: false
+      - name: createChangeStream
+        object: *collection0
+        arguments:
+          pipeline: []
+        saveResultAsEntity: &changeStream0 changeStream0
+      - name: insertOne
+        object: *globalCollection0
+        arguments:
+          document:
+            x: 1
+      - name: iterateUntilDocumentOrError
+        object: *changeStream0
+        expectResult:
+          _id:
+            $$exists: true
+          documentKey:
+            $$exists: true
+          operationType: insert
+          ns:
+            db: *database0
+            coll: *collection0
+          fullDocument:
+            x:
+              $numberInt: "1"
+            _id:
+              $$exists: true
+    expectEvents:
+      - client: *client0
+        ignoreExtraEvents: true
+        events:
+          - commandStartedEvent:
+              command:
+                aggregate: *collection0
+                cursor: {}
+                pipeline:
+                  - $changeStream: {}
+              commandName: aggregate
+              databaseName: *database0
+          - commandStartedEvent:
+              command:
+                getMore:
+                  $$exists: true
+                collection: *collection0
+              commandName: getMore
+              databaseName: *database0
+          - commandStartedEvent:
+              command:
+                aggregate: *collection0
+                cursor: {}
+                pipeline:
+                  - $changeStream:
+                      resumeAfter:
+                        $$unsetOrMatches:
+                          $$exists: true
+              commandName: aggregate
+              databaseName: *database0
+  - description: change stream resumes after StaleEpoch
+    runOnRequirements:
+      - minServerVersion: 4.3.1
+        topologies:
+          - replicaset
+          - sharded
+          - load-balanced
+    operations:
+      - name: failPoint
+        object: testRunner
+        arguments:
+          client: globalClient
+          failPoint:
+            configureFailPoint: failGetMoreAfterCursorCheckout
```

Comment:
There was a [link](https://github.com/mongodb/specifications/blob/master/source/change-streams/tests/legacy/change-streams-resume-errorLabels.yml#L9) to SERVER ticket here that might still be helpful to leave in.

Reply:
Derp, forgot to preserve comments in the conversion.  Fixed here and elsewhere.